io/net/virt-net/network_virtualization.py

```diff
@@ -603,6 +603,28 @@ def enable_auto_failover(self):
             self.fail("Could not enable auto failover")
         return True
 
+    def get_failover_priority(self, logport):
+        """
+        get the priority value for the given backing device
+        """
+        cmd = 'lshwres -r virtualio -m %s --rsubtype vnic --level lpar \
+               --filter lpar_names=%s -F slot_num,backing_devices' \
+               % (self.server, self.lpar)
+        try:
+            output = self.run_command(cmd)
+        except CommandFailed as cmdfail:
+            self.log.debug(str(cmdfail))
+            self.fail("lshwres operation failed ")
+        for backing_dev in output:
+            if backing_dev.startswith('%s,' % self.slot_num):
+                backing_dev = backing_dev.strip('%s,"' % self.slot_num)
+                for entry in backing_dev.split(','):
+                    entry = entry.split('/')
+                    if logport in entry:
+                        priority = entry[8]
+                        break
+        return priority
```

Comment:
Have a default value for priority, so that we use it for returning when not set.

Reply:
done